rpcd/playbooks/roles/rpc_maas/tasks/main.yml

```diff
@@ -66,10 +67,12 @@
   vars:
     external_vip_address: "{{ external_lb_vip_address }}"
   when: >
+    groups['swift_all']|length > 0 and
     inventory_hostname in groups['swift_all']
 
 - include: ceph.yml
   when: >
+    groups['ceph_all']|length > 0 and
     inventory_hostname in groups['ceph_all'] and
     groups['ceph_all'] is defined
```

Comment:
Shouldn't ``groups['ceph_all'] is defined`` be moved to be the first conditional in this expression?